stable/index.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+entries:
+  portworx:
+  - apiVersion: v1
+    created: "2020-07-06T05:21:12.004341091Z"
+    description: A Helm chart for installing Portworx on Kubernetes.
+    digest: 74dde97b5000b145904e6070eaa0ad16159ca8d8f846780c6934b20fffc6b774
+    home: https://portworx.com/
+    icon: https://raw.githubusercontent.com/portworx/helm/master/doc/media/px-logo.png
+    name: portworx
+    sources:
+    - https://github.com/portworx/helm
+    urls:
+    - https://raw.githubusercontent.com/tusharraut1994/helm/master/stable/portworx-1.0.0.tgz
```

Comment:
is this supposed to be pointing to `tusharraut1994` repo?

Reply:
No, I will change this to point to portworx repo.

Reply:
Like: https://raw.githubusercontent.com/portworx/helm/master/stable/portworx-1.0.0.tgz